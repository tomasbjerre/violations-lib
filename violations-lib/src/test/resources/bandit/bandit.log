/jenkins/workspace/ure_RC_SMTP_Bridge_flake8-bandit@2/rc_smtp_bridge/rc_smtp_bridge.py:42: MEDIUM: B307: Use of possibly insecure function - consider using safer ast.literal_eval.
