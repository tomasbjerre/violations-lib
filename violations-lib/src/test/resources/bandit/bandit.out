/home/bjerre/workspace/bandit/examples/assert.py:1: LOW: B101: Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.
/home/bjerre/workspace/bandit/examples/binding.py:4: MEDIUM: B104: Possible binding to all interfaces.
/home/bjerre/workspace/bandit/examples/cipher-modes.py:6: MEDIUM: B305: Use of insecure cipher mode cryptography.hazmat.primitives.ciphers.modes.ECB.
